demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -746,6 +748,18 @@ export class DemoMeetingApp
       );
     });
 
+    if(!this.defaultBrowserBehaviour.supportsCanvasCapturedStreamPlayback())  {
+      document.getElementById('video-input-filter-container').className += ' no-canvas-playback';
```

Comment:
Can we just set the style to none like other places in the code? Also do you need the check for unifiedplan like the other place in the code too?
```
if (
      this.defaultBrowserBehaviour.supportsCanvasCapturedStreamPlayback() &&
      this.enableUnifiedPlanForChromiumBasedBrowsers
    )
```

Reply:
I do not have to check for unified plan as far as I understand. The code snippet shared is the one used for checking for Segmentation processor. In my code, I am just checking whether the browser supports canvas capture playback. This check is done because iOS browsers will not be able to support video filters as they lack canvas capture stream playback.